src/gurt/telemetry.c

```diff
@@ -286,7 +286,7 @@ d_tm_init(int id, uint64_t mem_size, int flags)
 	}
 	*base_addr = (uint64_t)d_tm_shmem_root;
 
-	snprintf(tmp, sizeof(tmp), "ID: %d", id);
+	snprintf(tmp, sizeof(tmp), "I/O Engine %d", id);
```

Comment:
Depending on which of us lands first, one of us will need to fix the event processing in the prometheus exporter, to properly clean up the metric names.

Reply:
This is just a reminder rather than anything actionable right now. But the prometheus exporter in the control plane keys off this string to clean up metric names for display.

Reply:
I reverted it.